native/cocos/scene/Camera.cpp

```diff
@@ -57,6 +57,8 @@ constexpr ccstd::array<ccstd::array<float, 4>, 4> PRE_TRANSFORMS = {{
     {{0, -1, 1, 0}}   // SurfaceTransform.ROTATE_270
 }};
 
+std::atomic<uint32_t> sCameraId{0};
```

Comment:
Will it need to be defined as a `atomic` variable?
I think Camera could not be created in difference thead.